tools/release-ci/cmd/entrypoint.go

```diff
@@ -0,0 +1,138 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/release/tools/release-ci/pkg/logging"
+)
+
+// INTERNAL_ERROR is the error code we use to indicate
+// an internal error in the entrypoint occurred
+const INTERNAL_ERROR = 127
+
+// entrypointCmd runs the command given and tees output
+var entrypointCmd = &cobra.Command{
+	Use:   "entrypoint EXECUTABLE [ARGS...]",
+	Short: "Executes the command provided and copies output to a file",
+	Long: `Executes the command provided and copies output to a file
+
+The GCS sidecar container will run alongside the container that
+uses this command as an entrypoint. These two processes are
+configured together; the sidecar will look for the output from
+this entrypoint to upload to GCS.`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := runEntrypoint(args); err != nil {
+			fmt.Printf("Error: %v\n", err)
+			os.Exit(1)
+		}
+	},
+}
+
+var configurationFile string
```

Comment:
The package structure means _this_ `var` is used for both `entrypoint` and `sidecar` -- do we care?